### ISSUE TITLE
Missing 'ImportExpression' in visitorKeys

### ISSUE THREAD

**Issue by vilchik-elena** [babel/babel-eslint#837]
Hi,

When parsing with `babel-eslint` code containing `import()` I would expect `visitorKeys` to contain `ImportExpression`. Could you clarify why it is not the case? 

Using latest babel-eslint 10.1.0

```
const babel = require('babel-eslint');
const eslint = require('eslint');
const result = babel.parseForESLint("import('moduleName')");
console.log(result.visitorKeys['ImportExpression']); // prints 'undefined'
const wrapped = new eslint.SourceCode("import('moduleName')", result.ast);
console.log(wrapped.visitorKeys['ImportExpression']); // prints [ 'source' ]
```

Wrapping the result into `new SourceCode()` resets `visitorKeys` to some eslint's defaults which contain `ImportExpression`.

**Comment by JLHwung** [comment]
It is fixed in https://github.com/babel/babel/pull/10828, `babel-eslint` will be renamed as `@babel/eslint-parser`, for which we are still preparing.

**Comment by vilchik-elena** [comment]
@JLHwung thanks! does it mean that there is no yet release with the fix?

**Comment by JLHwung** [comment]
@vilchik-elena Yes, we can expect a release soon.

**Comment by JLHwung** [comment]
Fixed in `@babel/eslint-parser` 7.11.0, see https://babel.dev/blog/2020/07/13/the-state-of-babel-eslint#the-future for the renaming guide.
